pages/documentation/glossary.md

```diff
@@ -7,5 +7,10 @@ layout: default
 <span class="heading" id="exporter">**Exporter:**</span>  
 An exporter is a class responsible for converting an [IIDM]() model and writing it to a file.
 
-<span class="heading" id="importer">**Importer:**</span>  
+<span class="heading" id="importer">**Importer:**</span>
+<a id="importer"/>  
 An importer is a class responsible for loading a grid from a file and converting it to an [IIDM]() grid model.
+
+<span class="heading" id="tso">**TSO:**</span> 
+<a id="tso"/>  
+European Transmission System Operator is an entity operating independently from the other electricity market players and is responsible for the bulk transmission of electric power on the main high voltage electric networks.
```

Comment:
I would remove *European*: I think the TSO term is also valid outside Europe?

Reply:
Totally right, TSO is universal

Reply:
Removed.